package.json

```diff
@@ -37,7 +37,8 @@
     "clean": "lerna run --loglevel=silent clean",
     "build": "lerna run --loglevel=silent build",
     "build:current": "lerna run --loglevel=silent build:current",
-    "pretest": "npm run build:current",
+    "pretest": "npm run build:current && npm run cli:link",
+    "cli:link": "cd packages/cli && npm link",
```

Comment:
Why do we do this?

Reply:
I'm okay with the `cli:link` command, but not having it added to pretest. This means that any time someone runs tests, they're going to lose their existing global installation of the CLI and have it replaced with the locally-linked one. I expect this would cause havoc for contributors down the line.